gradle/test.gradle

```diff
@@ -173,6 +173,12 @@ gradle.taskGraph.whenReady({ graph ->
         jvmArgs += ["--add-opens", "java.xml/jdk.xml.internal=ALL-UNNAMED"]
         jvmArgs += ["--add-opens", "java.base/jdk.internal.module=ALL-UNNAMED"]
         jvmArgs += ["--add-opens", "java.base/java.lang.module=ALL-UNNAMED"]
+        jvmArgs += ["--add-opens", "java.base/jdk.internal.ref=ALL-UNNAMED"]
+        jvmArgs += ["--add-opens", "java.base/jdk.internal.reflect=ALL-UNNAMED"]
+        jvmArgs += ["--add-opens", "java.base/jdk.internal.loader=ALL-UNNAMED"]
+        jvmArgs += ["--add-opens", "java.base/jdk.internal.misc=ALL-UNNAMED"]
+        jvmArgs += ["--add-opens", "java.base/jdk.internal.util.jar=ALL-UNNAMED"]
+        jvmArgs += ["--add-opens", "jdk.management/com.sun.management.internal=ALL-UNNAMED"]
```

Comment:
if these add-opens are needed for test, might they be needed elsewhere like dunit/internal/ProcessManager or session/tests/ServerContainer?

Reply:
What's weird is each one of those things has a different set of add-opens than test.gradle, even before my changes. Should they be the same? In my case I just needed these for an integration test that is doing reflection to measure the memory usage of the cache.

Reply:
ahh, so this is test code that needs the add-opens (not product code under test).  looks good then.